.github/workflows/release.yaml

```diff
@@ -67,4 +67,17 @@ jobs:
           version: latest
           args: release --release-notes=docs/release_notes/${{ steps.prep.outputs.VERSION }}.md --skip-validate
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
\ No newline at end of file
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Get release doc content
+        id: vars
+        run: echo ::set-output name=release_notes::$(cat docs/release_notes/${{ steps.prep.outputs.VERSION }}.md)
+      - name: Notify slack of new release
+        env:
+          SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
```

Comment:
does this exist?

Reply:
https://github.com/weaveworks/corp/issues/1911

Reply:
shall we block merging on this? I'll pre-approve now.

Reply:
Yeah for sure. I'll wait until that exists.